tests/gui/test_DirectLabel.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+from direct.gui.DirectLabel import DirectLabel
+
+
+def test_create_label():
+    position = (0.0, 0.0, -0.6)
+    text = "hello"
+    scale = (0.1, 0.1, 0.1)
+    label = DirectLabel(text=text,
+                        pos=position,
+                        scale=scale,
+                        textMayChange=1)
```

Comment:
This test should at least test something to verify that the creation was successful (ie. test that it correctly applied the text, or something like that)